src/com/android/volley/toolbox/ImageLoader.java

```diff
@@ -165,6 +165,25 @@ public boolean isCached(String requestUrl, int maxWidth, int maxHeight) {
     }
 
     /**
+     * get Image for background caching
+     * @param requestUrl
+     * @return
+     */
+    public ImageContainer getImageBackground(String requestUrl) {
+        return get(requestUrl, new ImageListener() {
+            @Override
+            public void onResponse(ImageContainer response, boolean isImmediate) {
+
```

Comment:
comment why we have empty listener?
or maybe check for a non-null listener inside get()